src/main/java/marquez/service/JobService.java

```diff
@@ -210,6 +218,16 @@ public Job createOrUpdate(
         versions
             .labels(namespaceName.getValue(), jobMeta.getType().toString(), jobName.getValue())
             .inc();
+
+        if (jobMeta.getRunId().isPresent()) {
```

Comment:
You can avoid the `if` check and directly called `get()` on the optional by using [`ifPresent()`](https://docs.oracle.com/javase/8/docs/api/java/util/Optional.html#ifPresent-java.util.function.Consumer-):

```java
   jobMeta
     .getRunId()
     .ifPresent( ... )
```

We have a similar logic when [creating a job with a run ID](https://github.com/MarquezProject/marquez/blob/main/src/main/java/marquez/service/JobService.java#L219)